src/components/sdfgViewer.ts

```diff
@@ -312,7 +387,7 @@ export class SdfgViewerProvider
         // it from the open editors list.
         webviewPanel.onDidDispose(() => {
             SdfgViewerProvider.getInstance()?.removeOpenEditor(
-                webviewPanel.webview
+                document
```

Comment:
We can't access the webview here as it's already disposed and so the editor is never removed from the openEditors list. This is why we have to get the editor by the path instead of by the webview to remove it.
This fix removes the error that always occurs when closing an SDFG!

Reply:
Nice catch, thanks!